Xml.psm1

```diff
@@ -634,44 +635,48 @@ function ConvertFrom-Html {
     return $xml
 }
 
-
 function Convert-Xml {
     #.Synopsis
     #   The Convert-XML function lets you use Xslt to transform XML strings and documents.
     #.Description
     #   Documentation TODO
-    [CmdletBinding(DefaultParameterSetName="Xml")]
+    [CmdletBinding(DefaultParameterSetName="Xsl")]
```

Comment:
I felt Xsl vs. XslCompiledTransform conveyed more than Xml vs. XslCompiledTransform.  "Xml" seemed like a (probable cut and paste) carryover from other functions in the module.  But, the name is functionally insignificant.  If you prefer, leave it "Xml" or call it "JohnBigBooty".  What matters is that I've added an alternate, distinct parametersetname: "XslCompiledTransform".